README.md

```diff
@@ -227,6 +228,175 @@ For further configuration options (some have specific requirements about your en
 
 [Get in touch](https://wire.com/pricing/).
 
+## Monitoring
+
+### Installation
+
+The following instructions detail the installation of a monitoring system consisting
+of a Prometheus instance and corresponding Alert Manager in addition to a Grafana
+instance for viewing dashboards related to cluster and wire-services health.
+
+The monitoring system requires disk space if you wish to be resilient to pod
+failure. If you are deployed on AWS you may install the `aws-storage` helm
+chart which provides configurations of Storage Classes for AWS's elastic block
+storage (EBS).
+
+First we install the Storage Classes via the `aws-storage` chart:
+
+```
+helm upgrade --install --namespace demo demo-aws-storage charts/aws-storage \
+    -f values/wire-server/demo-values.yaml \
+    -f values/wire-server/demo-secrets.yaml \
```

Comment:
I added them on every command because at certain points I recommend adding overrides to your values file to tweak certain things; in general they're not necessary, but obviously if there is config in there that the user has tweaked they'll need to be applied. It doesn't cause any harm to leave them in there and can potentially avoid confusion from users who don't really know what they're doing.

I can certainly remove the secrets one, but I don't see any harm in passing the values file.